x/auth/client/cli/tx_multisign.go

```diff
@@ -29,6 +29,79 @@ type BroadcastReq struct {
 	Mode string         `json:"mode" yaml:"mode"`
 }
 
+// GetMultiMsgSignCommand returns the multi-msg-sign cmd
+func GetMultiMsgSignCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "multi-msg-sign [file] [file]...",
+		Short:   "Combine messages from transactions which are generated offline.",
+		PreRun:  preSignCmd,
+		Example: fmt.Sprintf("%s tx multi-msg-sign tx1.json tx2.json tx3.json ", version.AppName),
+		Long: strings.TrimSpace(
+			fmt.Sprintf(
+				`Combine the transactions created with the --generate-only flag and Sign those transactions.
+
+Example: 
+$ %s tx multi-msg-sign tx1.json tx2.json tx3.json 
+
+It will read a transactions from [files], sign it, and print its JSON encoding.
+
+The --offline flag makes sure that the client will not reach out to full node.
+As a result, the account and sequence number queries will not be performed and
+it is required to set such parameters manually. Note, invalid values will cause
+the transaction to fail.
+
+The --multisig=<multisig_key> flag generates a signature on behalf of a multisig account
+key. It implies --signature-only. Full multisig signed transactions may eventually
+be generated via the 'multisign' command.
+`, version.AppName,
+			)),
+
+		RunE: makeMultiMsgSignCmd(),
+	}
+
+	cmd.Flags().String(flagMultisig, "", "Address or key name of the multisig account on behalf of which the transaction shall be signed")
+	cmd.Flags().Bool(flagOverwrite, false, "Overwrite existing signatures with a new one. If disabled, new signature will be appended")
```

Comment:
Q: When would you not want to overwrite? When using a multi-sig, right?

Reply:
yes